static/js/components/Loading_test.js

```diff
@@ -1,134 +1,44 @@
-    shallow(<Loading restStates={restStates} renderContents={() => <div>CONTENT</div>} />)
+  const renderLoading = (loaded: boolean, errored: boolean) => {
+    return mount(<LoadingContent loaded={loaded} errored={errored} />)
+  }
 
   describe("should render loading correctly", () => {
-    it("single state with loaded:false", () => {
-      const wrapper = renderLoading([
-        {
-          loaded:     false,
-          processing: false
-        }
-      ])
-      assert.equal(wrapper.text(), "Loading")
-    })
-
-    it("multiple states with one loaded:false", () => {
-      const wrapper = renderLoading([
-        {
-          loaded:     true,
-          processing: false
-        },
-        {
-          loaded:     false,
-          processing: false
-        }
-      ])
-      assert.equal(wrapper.text(), "Loading")
-    })
-
-    it("single state with processing:true", () => {
-      const wrapper = renderLoading([
-        {
-          loaded:     false,
-          processing: true
-        }
-      ])
-      assert.equal(wrapper.text(), "Loading")
-    })
-
-    it("multiple states with one processing:true", () => {
-      const wrapper = renderLoading([
-        {
-          loaded:     false,
-          processing: false
-        },
-        {
-          loaded:     false,
-          processing: true
-        }
-      ])
+    it("render loading if not loaded and not errored", () => {
+      const wrapper = renderLoading(false, false)
       assert.equal(wrapper.text(), "Loading")
     })
   })
 
   describe("should render errors correctly", () => {
-    it("single state with error", () => {
-      const wrapper = renderLoading([
-        {
-          loaded:     true,
-          processing: false,
-          error:      "bad things"
-        }
-      ])
-      assert.equal(wrapper.text(), "Error loading page")
-    })
-
-    it("multiple states all with errors", () => {
-      const wrapper = renderLoading([
-        {
-          loaded:     true,
-          processing: false,
-          error:      "bad things"
-        },
-        {
-          loaded:     true,
-          processing: false,
-          error:      "bad things"
-        }
-      ])
+    it("if not loaded", () => {
+      const wrapper = renderLoading(false, true)
       assert.equal(wrapper.text(), "Error loading page")
     })
 
-    it("multiple states with one error", () => {
-      const wrapper = renderLoading([
-        {
-          loaded:     true,
-          processing: false,
-          error:      undefined
-        },
-        {
-          loaded:     true,
-          processing: false,
-          error:      "bad things"
-        }
-      ])
+    it("if loaded", () => {
+      const wrapper = renderLoading(true, true)
       assert.equal(wrapper.text(), "Error loading page")
     })
   })
 
-  describe("should the contents if no errors", () => {
+  describe("should the contents if no errors and loaded", () => {
```

Comment:
could this `describe` block just be an `it` block? the message I think is a bit strangely worded too, I would say `it('should show the contents if no errors and loaded', () => {})` or something